nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -628,6 +628,7 @@ sub generateNetworkingDhcpConfig {
         write_file($fn, <<EOF);
 @configuration@
 EOF
+        print STDERR "For more hardware-specific settings check out https://github.com/NixOS/nixos-hardware"
```

Comment:
=> `For more hardware-specific settings, see https://github.com/NixOS/nixos-hardware.`

Reply:
I skipped the trailing dot because people/terminal emulators might by accident copy it into browser.